euphonic/cli/optimise_dipole_parameter.py

```diff
@@ -0,0 +1,145 @@
+# -*- coding: UTF-8 -*-
+"""
```

Comment:
Noting this is the optimize_eta.py with eta->dipole_parameter. I think it is possible to coerce git into thinking that the rename is part of the git update rather than two separate ops, delete file and new file. Then github will show the compariison.

Reply:
I've tried doing the `git mv` and editing as two separate commits to try and get git to think it's just been edited, but it turns out this is limited by git's 'similarity index', which will automatically detect a file as new if its been changed by more than 50% - see [git diff docs](https://git-scm.com/docs/git-diff#Documentation/git-diff.txt--Mltngt).

If we do:
`git diff master 114_rename_eta --stat -M -- euphonic\cli\optimise_eta.py euphonic\cli\optimise_dipole_parameter.py`
if shows as a delete & new file, whereas if we manually set the threshold to 40% similarity with `-M40%`
`git diff master 114_rename_eta --stat -M40% -- euphonic\cli\optimise_eta.py euphonic\cli\optimise_dipole_parameter.py`

It shows as a single modified file. Interesting, but nothing can be done about it I don't think!